src/libs/examples/z3-sat-model-test.sml

```diff
@@ -41,7 +43,11 @@ fun produce_sat_thm term model =
 
 (* Get a SAT model using Z3 (this function assumes that the given term is SAT) *)
 val term = ``(z + y = 2 * x) /\ ((x * x + y - 25) = z:int)``;
+(* (FUN_MAP2 (K 0w) (UNIV)) *)
+(*
+val term = ``(m2 = FUPDATE (m1: word32 |-> word8) (3w, 2w)) /\ (FAPPLY m2 3w = 2w)``;
```

Comment:
Can you move this commented-out term below as not-commented code? This will enable using `xxx-test.sml` examples in the CI (maybe they already are, I don't know).

Reply:
I think we should do this exactly when we write the test.sml file